source/partials/next_release/_release.md.erb

```diff
@@ -1,37 +1,59 @@
-<h4>some feature</h4>
+<h4> New Pipeline Creation Wizard </h4>
 
-<h4>Versions of Java this release of GoCD works with</h4>
-<ul>
-  <li>Java 8 (Deprecated)</li>
-  <li>Java 10 (Deprecated)</li>
-  <li>Java 11</li>
-  <li>Java 12</li>
-</ul>
+We have introduced a new simplified pipeline creation wizard towards improving the user experience.
```

Comment:
We have introduced a new simplified pipeline creation wizard as a step towards improving the user experience?

Reply:
`UI refresh` or `UX refresh`?